src/NUnitEngine/nunit.engine.tests/Services/DefaultTestRunnerFactoryTests.cs

```diff
@@ -55,48 +55,22 @@ public void ServiceIsStarted()
             Assert.That(_factory.Status, Is.EqualTo(ServiceStatus.Started));
         }
 
+        //Note: These tests currently overlap with RunnerSelectionTests, and are being removed as they are
+        //re-implemented in that test class
```

Comment:
Least important thing imaginable- for consistency, you could add a space between `//` and the following text if you want to. :D